src/components/hmi_message_handler/src/websocket_session.cc

```diff
@@ -128,14 +130,15 @@ void WebsocketSession::Read(boost::system::error_code ec,
   std::string data = boost::beast::buffers_to_string(buffer_.data());
   m_receivingBuffer += data;
 
+  JsonReader reader;
   Json::Value root;
-  if (!m_reader.parse(m_receivingBuffer, root)) {
-    std::string str_err = "Invalid JSON Message: " + data;
-    LOG4CXX_ERROR(ws_logger_, str_err);
+
+  if (!reader.parse(data, &root)) {
+    LOG4CXX_ERROR(ws_logger_, "Invalid JSON Message.");
     return;
   }
 
-  std::string wmes = m_receiverWriter.write(root);
+  std::string wmes = Json::writeString(m_receiverWriter, root);
```

Comment:
@ydementieiev const?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)